### ISSUE TITLE
Laravel 5.2 compatibility

### ISSUE THREAD

**Issue by michaeljhopkins** [vluzrmos/laravel-slack-api#5]
Can you make the necessary changes to allow for Laravel 5.2? I believe it involves updating guzzle to 6.0. I attempted to update composer.json, however it yields this error now, and unfortunately I'm not familiar enough with Guzzle to know the correct corse of action

```
[Symfony\Component\Debug\Exception\FatalThrowableError]                                                                                                                                                           
  Type error: Argument 3 passed to GuzzleHttp\Client::request() must be of the type array, boolean given, called in /home/m/Projects/Web/SlackAgainstAsAService/vendor/guzzlehttp/guzzle/src/Client.php on line 87 
```

**Comment by skaisser** [comment]
Can't wait to see this solved. Ill only upgrade to 5.2 after this perfect package is compatible.

Great job

**Comment by vluzrmos** [comment]
@michaeljhopkins Laravel suggests you to install guzzle `~5.3|~6.0`, the first one is compatible with laravel-slack-api, so can you try to downgrade your guzzle to `~5.3` instead of `~5.3|~6.0` ? I can't test it right now...

**Comment by vluzrmos** [comment]
done!

**Comment by vluzrmos** [comment]
released v0.4.5 with support for 5.x
